src/lib/create-template.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
```

Comment:
Do we need this here? I don't think we'll be executing this file directly, right?